docs/administration/runner/runner-config.md

```diff
@@ -45,3 +56,8 @@ Example:
 ```
 java -Drunner.rundeck.overrideTempDir=true -Drunner.dirs.tmp=/your/custom/dir -jar runner.jar
 ```
+
+## Runner APIs
+
+[Runner APIs](/api#runner-management) are available to create, edit, download, and delete Runners.
```

Comment:
This link might need the `.md` to build properly.

Reply:
fixed